### ISSUE TITLE
Mean and STD values regarding our dataset is required

### ISSUE THREAD

**Issue by Nikronic** [Nikronic/CoarseNet#11]
We need to calculate `mean` and `std` of the whole dataset lazily (online) BEFORE applying `halftone`.

Actually, The output is something similar to ground truth and by the way we can see in the paper they have mentioned that ground truth is normalized `([-1, 1])`. And we cannot calculate mean and std of halftone because of random operations.

**Comment by Nikronic** [comment]
Please see commit https://github.com/Nikronic/CoarseNet/commit/a1a5300e3eaa5ca037e1385256a91176cf730139.
ref: https://discuss.pytorch.org/t/about-normalization-using-pre-trained-vgg16-networks/23560/9

Furthermore, for test purposes on out 9 image subset, here is the mean and std values:
`mean= [0.3918, 0.3725, 0.3191]`
`std=[0.4881, 0.4835, 0.4661]`

**Comment by Nikronic** [comment]
Please see commit https://github.com/Nikronic/CoarseNet/commit/81a229fb5b8e3753331b9ab98e3d64b6924bd48c .

**Comment by Nikronic** [comment]
How to deal with halftone if our images are normalized???
